test/integration_test.go

```diff
@@ -1465,6 +1465,28 @@ func TestApplyAllSkipFalse(t *testing.T) {
 	assert.NotContains(t, stderr, "Skipping terragrunt module")
 }
 
+func TestTerragruntInfo(t *testing.T) {
+	t.Parallel()
+
+	cleanupTerraformFolder(t, TEST_FIXTURE_HOOKS_INIT_ONCE_WITH_SOURCE_NO_BACKEND)
+	tmpEnvPath := copyEnvironment(t, "fixture-hooks/init-once")
+	rootPath := util.JoinPath(tmpEnvPath, TEST_FIXTURE_HOOKS_INIT_ONCE_WITH_SOURCE_NO_BACKEND)
+
+	showStdout := bytes.Buffer{}
+	showStderr := bytes.Buffer{}
+
+	err := runTerragruntCommand(t, fmt.Sprintf("terragrunt terragrunt-info --terragrunt-non-interactive --terragrunt-working-dir %s", rootPath), &showStdout, &showStderr)
+	logBufferContentsLineByLine(t, showStdout, "show stdout")
+
+	stdout := showStdout.String()
+
+	assert.Nil(t, err)
+	assert.Contains(t, stdout, "{")
+	assert.Contains(t, stdout, fmt.Sprintf(`"DownloadDir": "%s/%s"`, rootPath, TERRAGRUNT_CACHE))
+	assert.Contains(t, stdout, `"TerraformCommand": "terragrunt-info",`)
+	assert.Contains(t, stdout, "}")
```

Comment:
Could you `json.Unmarshall` the output and check it contains expected values?

Reply:
done.